dotCMS/src/main/java/com/dotmarketing/portlets/templates/design/bean/TemplateLayout.java

```diff
@@ -12,17 +12,17 @@
  */
 public class TemplateLayout {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a constructor to the class, or provide default values. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1258)